cristin-person/src/main/java/no/unit/nva/cristin/person/country/ListCountriesHandler.java

```diff
@@ -0,0 +1,45 @@
+package no.unit.nva.cristin.person.country;
+
+import static no.unit.nva.cristin.model.Constants.DEFAULT_RESPONSE_MEDIA_TYPES;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.google.common.net.MediaType;
+import java.net.HttpURLConnection;
+import java.util.List;
+import no.unit.nva.cristin.person.model.nva.Countries;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+
+@SuppressWarnings("unused")
+public class ListCountriesHandler extends ApiGatewayHandler<Void, Countries> {
+
+    private final ListCountriesApiClient apiClient;
+
+    public ListCountriesHandler() {
```

Comment:
We never return a single country value, only a list?

Reply:
yes, we only list the whole dataset. it is only 255 results and i see no reason to do a lookup. it is also not updated often. if we want to have a lookup i have to implement in cristin api as well.

Reply:
if you want a lookup i can add a lookup in a new PR?